src/pybind/mgr/cephadm/inventory.py

```diff
@@ -198,6 +203,8 @@ def load(self):
                 for d in j.get('devices', []):
                     self.devices[host].append(inventory.Device.from_json(d))
                 self.networks[host] = j.get('networks', {})
+                self.osdspec_previews[host] = j.get('osdspec_previews', [])
```

Comment:
```suggestion
                self.osdspec_previews[host] = j.get('osdspec_previews', {})
```
(according to `__init__`)